src/controllers/skills/skillsController.ts

```diff
@@ -55,22 +56,32 @@ export class SkillsController {
 		this.router.get('/content-management/skills', this.getSkills())
 		this.router.post('/content-management/skills', this.uploadAndProcess())
 		this.router.get('/content-management/skills/success', this.getSkillsSuccess())
+		this.router.get('/content-management/skills/generate-report', this.getSkillsReport())
+		this.router.get('/content-management/skills/add-new-question', this.getAddQuestion())
 	}
 
 	getSkills() {
 		return async (req: Request, res: Response, next: NextFunction) => {
-			let doesPolicyExist: boolean
-			await this.csrsService
-				.findByName('Policy')
-				.then(() => {
-					doesPolicyExist = true
-				})
-				.catch(() => {
-					doesPolicyExist = false
-				})
+			req.session!.save(() => {
```

Comment:
Don't we need the policy check to ensure the profession exists?

Reply:
Done in LC-466-profession-specific-page-logic (https://github.com/Civil-Service-Human-Resources/lpg-management/pull/250)